keras_core/backend/common/distribute_scope.py

```diff
@@ -0,0 +1,38 @@
+"""!!!DO NOT USE!!! Under construction and experiment."""
```

Comment:
Rename to `distribution.py`

Reply:
Ack, I will rename this after we settle on the name of the overall API.

Reply:
Done.